backend/benefit/calculator/api/v1/serializers.py

```diff
@@ -216,17 +216,28 @@ class PaySubsidySerializer(serializers.ModelSerializer):
 
     def _is_manual_mode(self):
         return (
-            self.context["request"].data["calculation"][
+            self.context["request"].data["calculation"] is not None
+            and self.context["request"].data["calculation"][
                 "override_monthly_benefit_amount"
             ]
             is not None
         )
 
+    HANDLING_STARTED_STATUSES = [
+        ApplicationStatus.HANDLING,
+        ApplicationStatus.ADDITIONAL_INFORMATION_NEEDED,
+        ApplicationStatus.ACCEPTED,
+        ApplicationStatus.REJECTED,
+    ]
+
+    def _is_handling_started(self):
```

Comment:
`_has_handling_started` instead of `_is_handling_started` would be better English but both are ok IMHO.